support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.tsx

```diff
@@ -406,7 +410,7 @@ function WeeklyCheckoutForm(props: PropTypes) {
 							validateForm={props.validateForm}
 							isTestUser={props.isTestUser}
 							setupRecurringPayPalPayment={props.setupRecurringPayPalPayment}
-							amount={props.price.price}
+							amount={props.discountedPrice.price}
```

Comment:
Have we tested this for products that don't have a promotion applied? Will they have a `discountedPrice`?

Reply:
Can we confirm we'd be billing the user with the non-discounted price once the promotion has come to an end?

Reply:
We have tested for single, monthly and yearly products with no discount. They come through as expected both in  Paypal summary at top right and in the mobile total shown.

Reply:
Can confirm in step function, support-workers-CODE, the CreateZuoraSubscription step creates two payments in the paymentSchedule, the discounted rate and the future full priced rate going forwards from a date in 3 months time. Have added screenshot at top here now too.

Reply:
@paul-daniel-dempsey @undergroundquizscene thanks for confirming this and providing evidence 👍